app/src/main/java/de/westnordost/streetcomplete/quests/cycleway/AddCyclewayForm.kt

```diff
@@ -34,11 +34,14 @@ import kotlinx.serialization.json.Json
 class AddCyclewayForm : AStreetSideSelectForm<CyclewayAndDirection, LeftAndRightCycleway>() {
 
     override val buttonPanelAnswers get() =
-        if (isDisplayingPrevious) listOf(
+        if (isDisplayingPrevious) {
+            listOf(
             AnswerItem(R.string.quest_generic_hasFeature_no) { isDisplayingPrevious = false },
             AnswerItem(R.string.quest_generic_hasFeature_yes) { onClickOk() }
         )
```

Comment:
Indentation:

```suggestion
        if (isDisplayingPrevious) {
            listOf(
                AnswerItem(R.string.quest_generic_hasFeature_no) { isDisplayingPrevious = false },
                AnswerItem(R.string.quest_generic_hasFeature_yes) { onClickOk() }
            )
```

Reply:
I mistakenly pushed this file. This is fixed with 46a24ef3c9d4190cffb92209cfd5e9317ce9ef7b.